### DISCUSSION TITLE
Locking Spacers for Horizontal Wall Control - Missing Files

### DISCUSSION THREAD

**Discussion by drbondi** [aderusha/DDD-Printable-Wall-Control-System#37] [General]
Hello, the files for 2x5 2x6 and 2x7 are missing or not posted.  Is it possible to get the posted?

They show in the image depicting the options, but there are no files listed for those 3 spacers.

Thank You!

**Comment by aderusha** [comment]
Those files should be there, are we talking about these same models?

https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Locking_spacer/2x5%20Locking%20Spacer.stl
https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Locking_spacer/2x6%20Locking%20Spacer.stl
https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Locking_spacer/2x7%20Locking%20Spacer.stl

**Reply by drbondi** [reply]
Thank you for the response, but no, those are the locking spacers for the vertical panels and I see where those are.

I am talking about the locking spacers for the Horizontal Wall Control panels.
